### PR TITLE
[MM-60859]  Replace FormattedMarkdownMessage in 'webapp/channels/src/components/admin_console/system_roles/system_role/system_role_permissions.tsx' with FormattedMessage

### PR DESCRIPTION
**Pull request by Aryakoste**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28695
Jira  https://mattermost.atlassian.net/browse/MM-60859
#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/admin_console/system_roles/system_role/system_role_permissions.tsx [hunk 1]
```diff
@@ -206,9 +206,21 @@ export default class SystemRolePermissions extends React.PureComponent<Props, St
...
+                <FormattedMessage
                     id='admin.permissions.roles.system_custom_group_admin.detail_text'
-                    defaultMessage="The built-in Custom Group Manager role can be used to delegate the administration of [Custom Groups](https://docs.mattermost.com/welcome/manage-custom-groups.html) to users other than the System Admin.\n \nDon't forget to deauthorize all other system users from administering Custom Groups by unchecking the associated permissions checkbox in System console > User Management > Permissions.\n \nThis role has permission to create, edit, and delete custom user groups by selecting **User groups** from the Products menu."
+                    defaultMessage="The built-in Custom Group Manager role can be used to delegate the administration of <a>Custom Groups</a> to users other than the System Admin.{br}{br}Don't forget to deauthorize all other system users from administering Custom Groups by unchecking the associated permissions checkbox in System console > User Management > Permissions.{br}{br}This role has permission to create, edit, and delete custom user groups by selecting <b>User groups</b> from the Products menu."
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/admin_console/system_roles/system_role/system_role_permissions.tsx:211]: Lets split these into p paragraphs
see https://github.com/mattermost/mattermost/pull/28438/files#r1791547386

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Comment by M-ZubairAhmed** [comment]
@Aryakoste let me know if you need help with this one

**Comment by Aryakoste** [comment]
@M-ZubairAhmed After every br I need to create a new p element right ?

**Comment by Aryakoste** [comment]
Or split the first two lines into one and third line.

**Comment by M-ZubairAhmed** [comment]
> @M-ZubairAhmed After every br I need to create a new p element right ?

Yes p will make them into single line, and the next p tag will start in new line

**Comment by Aryakoste** [comment]
@M-ZubairAhmed Changes done.

File: webapp/channels/src/components/admin_console/system_roles/system_role/system_role_permissions.tsx [hunk 2]
```diff
@@ -206,10 +206,40 @@ export default class SystemRolePermissions extends React.PureComponent<Props, St
...
+                <>
+                    <p>
+                        <FormattedMessage
+                            id='admin.permissions.roles.system_custom_group_admin.introduction'
+                            defaultMessage='The built-in Custom Group Manager role can be used to delegate the administration of <a>Custom Groups</a> to users other than the System Admin.'
+                            values={{
+                                a: (chunks: string) => (
+                                    <ExternalLink
+                                        href='https://docs.mattermost.com/welcome/manage-custom-groups.html'
+                                        location='manage-custom-groups'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/admin_console/system_roles/system_role/system_role_permissions.tsx:218]:
```suggestion
                                        location='adminConsoleSystemRoles'
```

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
Looks good just a small nit

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by larkox — APPROVED** [review]

**Comment by lindy65** [comment]
I've assigned to you for testing @Rutam21 - let's see if the test server builds - all checks have passed so it should

cc @yasserfaraazkhan

**Comment by Rutam21** [comment]
## This PR has been tested in the Test Server and the changes are working as expected.

### Current Server UI for Custom Group Manager Permission

Old-Enterprise_server

### Test Server Update for Custom Group Manager Permission

PR-Enterprise-Server

**There are no visual changes in the UI after the component update in this PR and the rich text formatting and functionality in this section are preserved as before.**

cc: @lindy65

**Comment by M-ZubairAhmed** [comment]
@Rutam21 thank you for testing this PR 🎉

**Comment by lindy65** [comment]
Thanks for testing @Rutam21 - counted towards Hacktoberfest

**Pull request merged**